code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -249,23 +252,12 @@
 				user.visible_message("<span class='warning'>[user]'s palms flare a bright red against [target]'s temples!</span>")
 				to_chat(target, "<span class='danger'>A terrible red light floods your mind. You collapse as conscious thought is wiped away.</span>")
 				target.Weaken(12)
-				sleep(20)
-				if(ismindshielded(target))
-					to_chat(user, "<span class='notice'>They have a mindshield implant. You begin to deactivate it - this will take some time.</span>")
-					user.visible_message("<span class='warning'>[user] pauses, then dips [user.p_their()] head in concentration!</span>")
-					to_chat(target, "<span class='boldannounce'>Your mindshield implant becomes hot as it comes under attack!</span>")
-					sleep(100) //10 seconds - not spawn() so the enthralling takes longer
-					to_chat(user, "<span class='notice'>The nanobots composing the mindshield implant have been rendered inert. Now to continue.</span>")
-					user.visible_message("<span class='warning'>[user] relaxes again.</span>")
-					for(var/obj/item/implant/mindshield/L in target)
-						if(L && L.implanted)
-							qdel(L)
-					to_chat(target, "<span class='boldannounce'>Your mental protection implant unexpectedly falters, dims, dies.</span>")
+				sleep(2 SECONDS)
```

Comment:
I think you should remove this sleep, now that nothing needs to happen after it, and because sleeps are evil. If you feel the need to compensate on the overall time it takes, you could probably change:
```dm
if(!do_mob(user, target, 7 SECONDS))
```
To
```dm
if(!do_mob(user, target, 7.7 SECONDS))
```
Or something like that. Making the total time to thrall ~23.1 seconds, just 0.1 over what it was before.

Reply:
Siding with steel. Remove this sleep.